pumpkin-lib/src/constraints/cumulative.rs

```diff
@@ -204,7 +203,7 @@ impl<Var: IntegerVariable + 'static> CumulativeConstraint<Var> {
     }
 }
 
-impl<Var: IntegerVariable + 'static> Constraint for CumulativeConstraint<Var> {
+impl<Var: IntegerVariable + 'static + Debug> Constraint for CumulativeConstraint<Var> {
```

Comment:
Not necessary but it was useful for debugging purposes, I will remove the bound (we could also consider globally adding it to the `IntegerVariable` trait since, currently, every implementation of the trait implements it)